includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -28,15 +28,24 @@ class WC_REST_Payments_Orders_Controller extends WC_Payments_REST_Controller {
 	 */
 	private $gateway;
 
+	/**
+	 * WC_Payments_Customer instance for working with customer information
+	 *
+	 * @var WC_Payments_Customer_Service
```

Comment:
Should be `WC_Payments_Customer_Service` here?

Reply:
Good catch! Thank you! Fixed.